server/functions.lua

```diff
@@ -436,7 +436,45 @@ exports('ExploitBan', ExploitBan)
 ---@return boolean
 function HasPrimaryGroup(source, filter)
     local playerData = QBX.Players[source].PlayerData
-    return HasPlayerGotGroup(filter, playerData)
+    return HasPlayerGotGroup(filter, playerData, true)
 end
 
 exports('HasPrimaryGroup', HasPrimaryGroup)
+
+---@param source Source
+---@param filter string | string[] | table<string, number>
+---@return boolean
+function HasGroup(source, filter)
+    local playerData = QBX.Players[source].PlayerData
+    return HasPlayerGotGroup(filter, playerData)
+end
+
+exports('HasGroup', HasGroup)
+
+---@param source Source
+---@return table<string, integer>
+function GetGroups(source)
+    local playerData = QBX.Players[source].PlayerData
+    return GetGroups(playerData)
+end
+
+exports('GetGroups', GetGroups)
+
+---@return PlayerData[]
+local function getPlayersData()
+    local playersData = {}
+    for _, player in pairs(QBX.Players) do
+        playersData[#playersData + 1] = player.PlayerData
+    end
+    return playersData
+end
+
+exports('GetPlayersData', getPlayersData)
```

Comment:
Do we really need this? We already have an export to get the QBX.Players, which is mostly PlayerData already.

Reply:
Doesn't this pass all the function tables as well which ox doesn't really need? I guess it's kind of extra